eng/Version.Details.xml

```diff
@@ -106,6 +106,14 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>7ee84596d92e178bce54c986df31ccc52479e772 </Sha>
     </Dependency>
+    <Dependency Name="System.Net.Http.WinHttpHandler" Version="4.7.0">
```

Comment:
dotnet/aspnetcore is behind on this version. Should we move to `4.7.2` (released in 3.1.5)❔

Reply:
Yes